docs-next/src/content/docs/interfaces/about.mdx

```diff
@@ -0,0 +1,7 @@
+---
+description: Mocha's available DSLs for writing tests.
+title: About Interfaces
+---
+
+Interfaces are how developers write tests to be executed by Mocha.
+Mocha's "interface" system allows developers to choose their style of DSL (Domain-Specific-Language).
```

Comment:
Generally I think it's written mostly sentence case, it's not a proper noun as much as a common idea :) ref https://en.wikipedia.org/wiki/Domain-specific_language

```suggestion
Mocha's "interface" system allows developers to choose their style of DSL (domain-specific language).
```